web/router.ex

```diff
@@ -17,6 +17,13 @@ defmodule ElixirBlog.Router do
     get "/", PostController, :index
 
     resources "/posts", PostController
+
+    get "/register", RegistrationController, :new
+    post "/register", RegistrationController, :create
+
+    get "/login", SessionController, :new
+    post "/login", SessionController, :create
+    get "/logout", SessionController, :delete
```

Comment:
Preferable logout should not be a get request. Use `link_to session_path(@conn, :delete), method: 'delete'`